### ISSUE TITLE
zlint.json: uknown rule names are not reported

### ISSUE THREAD

**Issue by DonIsaac** [DonIsaac/zlint#206]
**Version**
n/a
**Describe the bug**
Just had this trip a user up
```json
{
  "rules": {
    "supressed-errors": "warn"
  }
}
```
Note that `supressed` should be `suppressed`

**To Reproduce**
^^^

**Expected behavior**
Zlint should emit a warning or error about the unknown rule

**Actual behavior**
Nothing is printed and the rule remains disabled
